plugins/woocommerce/includes/abstracts/abstract-wc-payment-gateway.php

```diff
@@ -394,13 +393,23 @@ public function validate_fields() {
 	}
 
 	/**
-	 * If There are no payment fields show the description if set.
-	 * Override this in your gateway if you have some.
+	 * Default payment fields display. Override this in your gateway to customize displayed fields.
+	 *
+	 * By default this renders the payment gateway description.
+	 *
+	 * @since 1.5.7
```

Comment:
would be nice to indicate the change of behavior here, as a help to people who aren't reviewing the `git` source and blame.

```php
 * @since {NEXT_VERSION} Runs content through `wp_kses_post()` unless an
 *                       extension has modified the description already.
```

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/48432/commits/e555e3b3d5914e9b8d42d147522b61b70f9c10fe